tasks/get-plugins.yml

```diff
@@ -7,9 +9,17 @@
     owner: root
     group: root
 
+- set_fact:
+    plugin_url: "{{ cj_nexus_url }}/{{ cj_nexus_mirror }}"
+  when: cj_nexus
+
 - name: "Get plugin to local cache dir"
   get_url:
-    url: "http://mirrors.jenkins-ci.org/plugins/{{ item.name }}/{{ item.version }}/{{ item.name }}.{{ item.extention | d('hpi') }}"
+    url: "{{ plugin_url |d('http://mirrors.jenkins-ci.org') }}/plugins/{{ item.name }}/{{ item.version }}/{{ item.name }}.{{ item.extention | d('hpi') }}"
```

Comment:
Same as my previous review the prefix should be the role name - what is plugin_url !?